src/bin/mdbook-xgettext.rs

```diff
@@ -17,16 +17,16 @@
 //! This program works like `xgettext`, meaning it will extract
 //! translatable strings from your book. The strings are saved in a
 //! GNU Gettext `messages.pot` file in your build directory (typically
-//! `po/messages.pot`).
+//! `po/messages.pot`). When the `depth` parameter is included, a new
+//! directory will contain the template files split based on the tiers
+//! of directory nesting.
 
 use anyhow::{anyhow, Context};
 use mdbook::renderer::RenderContext;
-use mdbook::BookItem;
+use mdbook::{book::Chapter, BookItem};
```

Comment:
More of a FYI: If VS Code doesn't format this as

```suggestion
use mdbook::BookItem;
use mdbook::book::Chapter;
```

then I think it means that you are not using `rustfmt` from the nightly Rust channel. Try `rustup default nightly` and see if the formatting changes. The `rustfmt.toml` file is controlling this and we use a currently unstable option (which non-nightly `rustfmt`s will ignore with a warning).

The `dprint check` check in CI will use nightly Rust and should flag this as well (after you resolve the merge conflict — just rebase locally and throw away your old `Cargo.lock` file, it'll be regenerated when you run `cargo build`).